packages/appeals-service-api/src/services/back-office.service.js

```diff
@@ -21,43 +22,42 @@ class BackOfficeService {
+		await this.#backOfficeRepository.saveAppealForSubmission(appealContactDetails, appeal_id, appealDocumentIds)
+	}
 
-		let appealToSubmitToBackOffice = await getAppeal(id);
-		logger.debug(`Appeal found in repository: ${JSON.stringify(appealToSubmitToBackOffice)}`);
-		if (appealToSubmitToBackOffice === null) {
-			throw ApiError.appealNotFound(id);
-		}
+	async submitAppeals() {
+		const backOfficeSubmissions = await this.#backOfficeRepository.getAppealsForSubmission();
+		for (const backOfficeSubmission of backOfficeSubmissions) {
+			const id = backOfficeSubmission.getAppealId()
+			logger.debug(`Attempting to submit appeal with ID ${id} to the back office`);
+
+			let appealToSubmitToBackOffice = await getAppeal(id);
+			logger.debug(`Appeal found in repository: ${JSON.stringify(appealToSubmitToBackOffice)}`);
+
+			if (appealToSubmitToBackOffice.horizonId == undefined || appealToSubmitToBackOffice.horizonId == false) {
+				let appealAfterSubmissionToBackOffice;
+				if (isFeatureActive('send-appeal-direct-to-horizon-wrapper')) {
+					logger.debug('Using direct Horizon integration');
+					const horizonCaseReference = await this.#horizonService.submitAppeal(appealToSubmitToBackOffice);
+					appealAfterSubmissionToBackOffice = await saveAppealAsSubmittedToBackOffice(appealToSubmitToBackOffice, horizonCaseReference);
+				} else {
+					logger.debug('Using message queue integration');
+					appealAfterSubmissionToBackOffice = await saveAppealAsSubmittedToBackOffice(appealToSubmitToBackOffice);
+					this.#backOfficeRepository.create(appealAfterSubmissionToBackOffice);
+				}
 
-		if (appealToSubmitToBackOffice.horizonId == undefined || appealToSubmitToBackOffice.horizonId == false) {
-			let appealAfterSubmissionToBackOffice;
-			if (isFeatureActive('send-appeal-direct-to-horizon-wrapper')) {
-				logger.debug('Using direct Horizon integration');
-				const horizonCaseReference = await this.#horizonService.createAppeal(appealToSubmitToBackOffice);
-				appealAfterSubmissionToBackOffice = await saveAppealAsSubmittedToBackOffice(appealToSubmitToBackOffice, horizonCaseReference);
-			} else {
-				logger.debug('Using message queue integration');
-				appealAfterSubmissionToBackOffice = await saveAppealAsSubmittedToBackOffice(appealToSubmitToBackOffice);
-				this.#backOfficeRepository.create(appealAfterSubmissionToBackOffice);
+				await sendSubmissionConfirmationEmailToAppellant(appealAfterSubmissionToBackOffice);
+				await sendSubmissionReceivedEmailToLpa(appealAfterSubmissionToBackOffice);
 			}
 
-			await sendSubmissionConfirmationEmailToAppellant(appealAfterSubmissionToBackOffice);
-			await sendSubmissionReceivedEmailToLpa(appealAfterSubmissionToBackOffice);
-			return appealAfterSubmissionToBackOffice;
+			logger.debug('Appeal has already been submitted to the back-office');
 		}
 
-		logger.debug('Appeal has already been submitted to the back-office');
-		throw ApiError.appealAlreadySubmitted();
+		return;
```

Comment:
Isn't your IDE complaining about an unnecessary return in the last line?  A void return in the last line is redundant.

Reply:
it isn't! However, this can be removed if its offensive to people's sensibilities 😂